packages/radix-ui-themes/src/components/radio-card-group.tsx

```diff
@@ -23,7 +23,7 @@ const RadioCardGroupRoot = React.forwardRef<RadioCardGroupRootElement, RadioCard
       marginPropDefs
     );
     return (
-      <Grid asChild>
+      <Grid asChild columns={props.columns ?? 'repeat(auto-fit, minmax(160px, 1fr))'}>
```

Comment:
Should we do this through the prop defs?

Reply:
In `radio-card-group.props.ts`

Reply:
But these don't exist there…